samples/notebooks/Lightweight Python components - basics.ipynb

```diff
@@ -114,8 +114,12 @@
     "\n",
     "    (quotient, remainder) = divmod_helper(dividend, divisor)\n",
     "\n",
-    "    from tensorflow.python.lib.io import file_io\n",
     "    import json\n",
+    "    from pathlib import Path\n",
+    "\n",
+    "    #FIX: Metadata and metrics should be output like all other outputs\n",
+    "    ui_metadata_output_path='/mlpipeline-ui-metadata.json'\n",
```

Comment:
How do you make it work when user run the notebook in jupyter when the path is under root?